src/ome.ts

```diff
@@ -78,19 +78,27 @@ export async function loadOMEWell(config: ImageLayerConfig, store: HTTPStore, ro
     sourceData.name = `Well ${row}${col}`;
     sourceData.rows = Math.ceil(imagePaths.length / cols);
     sourceData.columns = cols;
-    sourceData.onClick = config.onClick || ((info: any) => {
+    sourceData.onClick = (info: any) => {
         let gridCoord = info.gridCoord;
         if (!gridCoord) {
             return;
         }
         const { row, column } = gridCoord;
         const { source } = sourceData;
+        let imgSource = undefined;
         if (typeof source === 'string' && !isNaN(row) && !isNaN(column)) {
             const field = (row * cols) + column;
-            let imgSource = join(source, imagePaths[field]);
+            imgSource = join(source, imagePaths[field]);
+        }
+        if (config.onClick) {
```

Comment:
Could you leave a comment and reference this PR for this block. I want to be explicit about what things we are doing that are ImJoy specific.

Reply:
Ah sorry, I already merged... I'll make sure to add a comment.